conda/_vendor/auxlib/entity.py

```diff
@@ -573,10 +573,10 @@ class ListField(Field):
     _type = tuple
 
     def __init__(self, element_type, default=NULL, required=True, validation=None,
-                 in_dump=True, default_in_dump=True, nullable=False, immutable=False):
+                 in_dump=True, default_in_dump=True, nullable=False, immutable=False, aliases=()):
         self._element_type = element_type
         super(ListField, self).__init__(default, required, validation,
```

Comment:
I think we should be using explicit named arguments

Reply:
What do you mean?

Reply:
```
 super(ListField, self).__init__(default, required, validation,
```

vs

```
 super(ListField, self).__init__(default, required=required, validation=validation,
```

?

Reply:
If @goanpeca means keyword-only arguments, then I don't know, they look ugly when you have to support Python 2 and don't have https://www.python.org/dev/peps/pep-3102/ ...